assets/styles/_base.scss

```diff
@@ -536,6 +539,25 @@ select {
 * Global layout classes for a consistent appearance.
 */
 
+
+.pageWrapper {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  .admin-bar & {
+    padding-top: 46px;
+
+    @media (min-width: $breakpoint-tablet + 14) {
```

Comment:
Please change the value to px, as it doesn't relate to our tablet breakpoint.

Reply:
Only needed if we keep the custom approach instead of the default WordPress margin approach.